api/modelmanager/modelmanager_test.go

```diff
@@ -418,8 +418,8 @@ func (s *modelmanagerSuite) TestListModelsWithInfo(c *gc.C) {
 		OwnerTag:           "user-admin",
 		Life:               params.Alive,
 		Status:             params.EntityStatus{Status: status.Status("active")},
-		Users:              []params.ModelUserInfo{},
-		Machines:           []params.ModelMachineInfo{},
+		User:               params.ModelUserInfo{},
```

Comment:
User probably needs to change, as we only have the Access and LastConnection information, none of the CreatedBy, etc. info.